spec/models/piece_spec.rb

```diff
@@ -15,7 +15,11 @@
         password: 'foobar',
         password_confirmation: 'foobar'
       )
-      @game = @user.games.create(white_player_id: @user)
+      @game = @user.games.create(white_player_id: @user.id)
+      FROM_ROW = 3
```

Comment:
These constants should be moved up between lines 10 and 11 (after the `describe` and before the `before`) as since they're in the `before(:all)`, Ruby technically attempts to redefine them before each test below, and you're technically not supposed to redefine constants ;)